frame/staking/runtime-api/Cargo.toml

```diff
@@ -15,10 +15,14 @@ targets = ["x86_64-unknown-linux-gnu"]
 [dependencies]
 codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive"] }
 sp-api = { version = "4.0.0-dev", default-features = false, path = "../../../primitives/api" }
+sp-staking = { version = "4.0.0-dev", default-features = false, path = "../../../primitives/staking" }
+pallet-staking = { version = "4.0.0-dev", default-features = false, path = "../../staking" }
```

Comment:
It is not a strict need, but technically we should move `Exposure`, or a type similar to it into `sp_staking` and not import the whole pallet staking here.

This API is supposed to be unbiased.